components/core/src/streaming_archive/writer/Archive.hpp

```diff
@@ -3,12 +3,14 @@
 
 // C++ libraries
 #include <memory>
+#include <optional>
 #include <set>
 #include <string>
 #include <unordered_set>
 #include <vector>
 #include <filesystem>
```

Comment:
```suggestion
#include <filesystem>
#include <memory>
#include <optional>
#include <set>
#include <string>
#include <unordered_set>
#include <vector>
```

might as well alphabetize the old code too

Reply:
-Fixed library order
-**All unit tests passed with current commits**
![image](https://github.com/y-scope/clp/assets/116512923/13493aad-c0b1-4870-afd9-e1bc045b7416)